kinds/structured/team/team_kind.cue

```diff
@@ -0,0 +1,31 @@
+package kind
+
+name: "Team"
+maturity: "merged"
+
+lineage: seqs: [
+	{
+		schemas: [
+			// v0.0
+			{
+				// The ID of an organisation the team belongs to.
```

Comment:
I think that all comments should start with the field name to make them consistent. 

```suggestion
				// OrgId is the ID of an organisation the team belongs to.
```

Reply:
i agree, i think this is a good guideline.

It's another thing we need guidelines on - what parts of schemas need commenting, and how to write good comments. I've started [a new section in the zero to maturity doc](https://docs.google.com/document/d/1gPzSCQn-0Qg86PYpPcR-l_kTQ4Ba1wgnNLZFKPSe8yQ/edit#heading=h.yrnsiig7os8c) to cover this.

Reply:
Updated the comments. Thanks, Selene, for a suggestion.